src/platform/site-wide/header/helpers/index.js

```diff
@@ -29,6 +29,12 @@ export const showLegacyHeader = () => {
   if (!legacyHeader) {
     return;
   }
+  const signInVis = window.location.pathname === '/sign-in/';
+
+  if (signInVis) {
+    const line = document.querySelector('.menu-rule.usa-one-whole');
+    line.style.borderBottom = 'none';
+  }
```

Comment:
No. Let's use CSS in our SignInApp to update styles

Reply:
updated